wgpu-hal/src/noop/mod.rs

```diff
@@ -140,6 +140,7 @@ pub fn adapter_info() -> wgt::AdapterInfo {
         driver: String::from("wgpu"),
         driver_info: String::new(),
         backend: wgt::Backend::Noop,
+        supports_transient: true,
```

Comment:
```suggestion
        supports_transient: false,
```

See the change in the doc comment on this field for the doc comment

Reply:
This needs to be updated here too